_posts/2023-10-09-Announcing-Data-Prepper-2.5.0.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.5.0"
+authors:
+- graytaylor0
+- oeyh
+date: 2023-10-09 11:30:00 -0500
+categories:
+  - releases
+meta_keywords: Data Prepper, OpenSearch migration, Processors, OpenSearch ingestion
+meta_description: OpenSearch has launched Data Prepper 2.5.0 with feature support for an OpenSearch source, translate and dissect processors, and key-value processor enhancements
+---
+
+Data Prepper 2.5.0 is now available for download. This release includes a new OpenSearch source, new dissect and translate processors, and additions to the existing key-value processor.
+
+## OpenSearch source
+
+The OpenSearch source is tailored for data migration and replication of OpenSearch clusters. While this is commonly done with [Snapshots](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/index/), there are often incompatibilities between snapshots of different versions within or between OpenSearch and Elasticsearch. In combination with Data Prepper’s [OpenSearch sink plugin](https://opensearch.org/docs/latest/data-prepper/pipelines/configuration/sinks/opensearch/), Data Prepper can now migrate all indices, or just specific indices, from one or more source clusters, to one or more sink clusters. The OpenSearch source will continually detect new indices in the source cluster that need to be processed, and can even be scheduled to reprocess indices at a configurable interval to pick up on new documents.
```

Comment:
Instead of the phrasal verb "pick up on", is there a more precise word we can use?

Reply:
just "process" would work